packages/cli/src/credentials/oauth2Credential.api.ts

```diff
@@ -130,7 +131,7 @@ oauth2CredentialController.get(
 			state: stateEncodedStr,
 		};
 
-		await Container.get(ExternalHooks).run('oauth2.authenticate', [oAuthOptions]);
```

Comment:
any reasons to change this line?

Reply:
Hi @netroy 
I have changed it. please have look.